zookeeper-api/src/main/java/org/apache/helix/zookeeper/routing/RoutingDataManager.java

```diff
@@ -175,6 +178,15 @@ public synchronized void reset() {
     _lastResetTimestamp = System.currentTimeMillis();
   }
 
+  public synchronized void reset(boolean enforceThrottle) {
```

Comment:
Do you mean forceReset? enforceThrottle is hard to understand.

Reply:
This one is "resetWithThrottle", the above one is technically "forceReset". I didn't want to change it and inflate PR size, but I'm leaning towards changing it now. I can 
1. Change it to be `reset(false)` and make `reset()` private; 
2. or change it to be `forceReset()`.